4-1_Cloud_Deployment.ipynb

```diff
@@ -0,0 +1,392 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Lesson 4.1: Deployment in the Cloud with Seldon & Kubeflow on AWS\n",
+    "\n",
+    "[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/zenml-io/zenbytes/blob/main/4-1_Cloud_Deployment.ipynb)\n",
```

Comment:
Not sure this one will run on colab, did you test it? The files will most probably not be present there?

Reply:
For this specific notebook I am not sure if we can get it to work on Colab at all tbh. For the code, we could probably get it onto Colab with git clone, but you also need stuff like AWS CLI, Docker, Kubeflow, ... installed

Reply:
Glad to have kicked off all these thoughts, looking forward to the changes :)